src/System.Runtime.Extensions/tests/System/IO/PathTests.cs

```diff
@@ -600,6 +602,24 @@ public static void GetFullPath_Windows_StrangeButLegalPaths()
         [InlineData(@"\\.\C:\Foo2\..")]
         public static void GetFullPath_Windows_ValidExtendedPaths(string path)
         {
+            if (PathFeatures.IsUsingLegacyPathNormalization())
+            {
+                // Legacy Path doesn't support any of these paths.
```

Comment:
Yes that is why I had to this ugly thing because they didn't share a parent. 

So you are suggesting that I should add it in AssertExtensions?